### DISCUSSION TITLE
How can I use the created TXT from `am -f > am-list.txt` to install all those applications on another system?

### DISCUSSION THREAD

**Discussion by Martin15135215** [ivan-hc/AM#76] [Q&A]
How can I use the created TXT from `am -f > am-list.txt` to install all those applications on another system? Because I want to do a clean installation of a new OS on my hardware and I would like to install all the applications I did before with the help from `am` on my new system.

**Comment by ivan-hc** [comment]
After you have installed AM again, open a terminal into the directory of the TXT file, then:
```
am -i $(echo $(cat am-list.txt | grep "-" | grep -v " am " | grep -o -P '(?<= ◆ ).*(?= - )'))
```
in case the new distro is an immutable one (as I have already said [here](https://github.com/ivan-hc/AM-Application-Manager#note-for-users-of-immutable-distributions)), do:
```
am --user
```
and follow the instructions if you want to keep this permanently, else:
```
alias am=/opt/am/appman
```
just to install the apps in this section (remember that you should use the alias above again in case you want to remove the apps).

And finally:
```
am -i $(echo $(cat am-list.txt | grep "-" | grep -v " am " | grep -o -P '(?<= ◆ ).*(?= - )'))
```
If you have other doubts, feerl free to ask.
